stable/hazelcast/values.yaml

```diff
@@ -0,0 +1,113 @@
+
+# Cluster settings
+cluster:
+  # memberCount is the number Hazelcast members
+  memberCount: 2
+
+# Hazelcast properties
+hazelcast:
+  # rest is a flag used to enable REST endpoints for Hazelcast member
+  rest: true
+  # javaOpts are additional JAVA_OPTS properties for Hazelcast member
+  javaOpts:
+  # configurationFiles are Hazelcast configuration files
+  configurationFiles:
+    hazelcast.xml: |-
+      <?xml version="1.0" encoding="UTF-8"?>
+      <hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.10.xsd"
+                     xmlns="http://www.hazelcast.com/schema/config"
+                     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+        <properties>
+          <property name="hazelcast.discovery.enabled">true</property>
+        </properties>
+        <network>
+          <join>
+            <multicast enabled="false"/>
+            <tcp-ip enabled="false" />
+            <discovery-strategies>
+              <discovery-strategy enabled="true" class="com.hazelcast.kubernetes.HazelcastKubernetesDiscoveryStrategy">
+                <properties>
+                  <property name="service-name">${serviceName}</property>
+                </properties>
+              </discovery-strategy>
+            </discovery-strategies>
+          </join>
+        </network>
+      </hazelcast>
+
+
+# nodeSelector is an array of Hazelcast Node labels for POD assignments
+# ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+nodeSelector: {}
+
+# Hazelcast Liveness probe
+livenessProbe:
+  # enabled is a flag to used to enable liveness probe
+  enabled: true
+  # initialDelaySeconds is a delay before liveness probe is initiated
+  initialDelaySeconds: 30
+  # periodSeconds decides how often to perform the probe
+  periodSeconds: 10
+  # timeoutSeconds decides when the probe times out
+  timeoutSeconds: 5
+  # successThreshold is the minimum consecutive successes for the probe to be considered successful after having failed
+  successThreshold: 1
+  # failureThreshold is the minimum consecutive failures for the probe to be considered failed after having succeeded
+  failureThreshold: 3
+
+# Hazelcast Readiness probe
+readinessProbe:
+  # enabled is a flag to used to enable readiness probe
+  enabled: true
+  # initialDelaySeconds is a delay before readiness probe is initiated
+  initialDelaySeconds: 30
+  # periodSeconds decides how often to perform the probe
+  periodSeconds: 10
+  # timeoutSeconds decides when the probe times out
+  timeoutSeconds: 1
+  # successThreshold is the minimum consecutive successes for the probe to be considered successful after having failed
+  successThreshold: 1
+  # failureThreshold is the minimum consecutive failures for the probe to be considered failed after having succeeded
+  failureThreshold: 3
+
+# Configure resource requests and limits
+# ref: http://kubernetes.io/docs/user-guide/compute-resources/
+#
+resources:
```

Comment:
Don't specify defaults for resources. Comment them out.

Reply:
Done